lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable automatic status emails for ICRV attempts
+    'ENABLE_ICRV_STATUS_EMAILS': False,
```

Comment:
Commit 88a537b0655153cba855b466364917dafdbb978b has the requisite Waffle changes.